### DISCUSSION TITLE
Copilot Chat menu is  disabled

### DISCUSSION THREAD

**Discussion by taosolutions-ca** [community/community#75452] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

How can I use the chat ? 

![image](https://github.com/community/community/assets/100535881/1868a5a4-ca0c-453a-b7be-cfb46b6bfb56)

Also when I sign in to github, it asks me to to sign up for github copilot chat private preview. But that page leads me to an invalid page:
https://github.com/github-copilot/chat_waitlist_signup/join

**Comment by davecheney** [comment, marked as answer]
@taosolutions-ca chat has been disabled by your organisation's administrator. please contact them directly.
